spec/replace-empty-node-spec.js

```diff
@@ -0,0 +1,289 @@
+var tidy = require('../');
+
+describe('Replacing empty dom nodes with containing tags', function() {
```

Comment:
Please also add tests for common cases

```html
<p>{{#tag}}</p>
<p>Hello world</p>
<p>{{/tag}}</p>
```

should become `{{#tag}}<p>Hello world</p>{{/tag}}`

and 

```html
<p>{{#tag}}</p>
<p>Hello world</p>
<p>{{/tag}} {{^tag}}</p>
<p>Good night</p>
<p>{{/tag}}</p>
```
should become `{{#tag}}<p>Hello world</p>{{/tag}} {{^tag}}<p>Good night</p>{{/tag}}`